.github/workflows/ui_publish.yml

```diff
@@ -0,0 +1,28 @@
+name: UI Publish
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish_app_web_staging:
+    runs-on: ubuntu-latest
+    env:
+      FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 14.16
+
+    - name: Install firebase
+      run: npm install firebase-tools
```

Comment:
why this one is npm and the other one is yarn?

Reply:
copy pasted from Itais deploy